content/id/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Kelola Sertifikat TLS Pada Klaster
+content_template: templates/task
+reviewers:
+- girikuncoro
+---
+
+{{% capture overview %}}
+
+Kubernetes menyediakan API `certificates.k8s.io` yang memungkinkan Kamu membuat sertifikat
+TLS yang ditandatangani oleh Otoritas Sertifikat (CA) yang Kamu kendalikan. CA dan sertifikat ini 
+bisa digunakan oleh _workload_ untuk membangun kepercayaan.
+
+API `certificates.k8s.io` menggunakan protokol yang mirip dengan [konsep ACME](https://github.com/ietf-wg-acme/acme/).
+
+{{< note >}}
+Certificates created using the `certificates.k8s.io` API are signed by a
+dedicated CA. It is possible to configure your cluster to use the cluster root
+CA for this purpose, but you should never rely on this. Do not assume that
+these certificates will validate against the cluster root CA.
+
```

Comment:
this can be removed since you have provided the translated paragraph below

Reply:
lol sorry. forget to delete